packages/neo-one-server-plugin-project/src/types.ts

```diff
@@ -52,9 +56,14 @@ export const projectConfigSchema = {
     },
   },
   codegen: {
```

Comment:
This should be:
```
codegen: {
  language: {
    format: ['javascript', 'typescript'],
    default: 'typescript',
  },
  framework: {
    format: ['none', 'react'],
    framework: 'none',
  },
}
```

See https://github.com/mozilla/node-convict. (Find this out on your own by seeing where `projectConfigSchema` is used).